python/py-serial/Portfile

```diff
@@ -49,3 +49,6 @@ if {${name} ne ${subport}} {
 
     livecheck.type  none
 }
+
+# Only match numeric versions (not betas)
+livecheck.regex     {archive/refs/tags/v([0-9.]+)\.tar\.gz}
```

Comment:
since you're using the `github` PG the usual way to do this is to set:
```github.livecheck.regex {([0-9.]+)}``` instead of re-implementing the `livecheck.regex`

Reply:
Ah, OK, that seems cleaner.  I didn't dig into the `github` PG, but just tweaked the regex as displayed by -d.

Force-pushed twice - once for the rebase and once for the change ( to simplify the diff).